analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AsyncVoidMethod.CSharp9.cs

```diff
@@ -117,4 +117,17 @@ private async void EventHandlerCases_MyEvent(object sender, bool e)
             await Task.Run(() => Console.WriteLine("test"));
         }
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/5432
+    public class Reproducer5432
+    {
+        public void SomeMethod()
+        {
+            var _timer = new System.Threading.Timer(RunOnceAsync);
+        }
+
+        private async void RunOnceAsync(object? _) // Noncompliant FP
```

Comment:
The nullability is not needed. And then this doesn't need to be in C# 9 file, but rather the general one for all languages.
```suggestion
        private async void RunOnceAsync(object _) // Noncompliant FP
```